x-pack/test/pki_api_integration/config.ts

```diff
@@ -54,8 +54,8 @@ export default async function({ readConfigFile }: FtrConfigProviderContext) {
       serverArgs: [
         ...xPackAPITestsConfig.get('kbnTestServer.serverArgs'),
         '--server.ssl.enabled=true',
-        `--server.ssl.key=${ES_KEY_PATH}`,
-        `--server.ssl.certificate=${ES_CERT_PATH}`,
+        `--server.ssl.key=${KBN_KEY_PATH}`,
+        `--server.ssl.certificate=${KBN_CERT_PATH}`,
```

Comment:
`@kbn/dev-utils` now provides a separate certificate/key for Kibana, switched to those to avoid confusion.